app/forms/providers/means/identify_income_form.rb

```diff
@@ -0,0 +1,100 @@
+module Providers
+  module Means
+    class IdentifyIncomeForm
+      include ActiveModel::Model
+
+      INCOME_TYPES = %i[benefits friends_or_family maintenance_in property_or_lodger pension].freeze
```

Comment:
Yeah that makes sense, thanks!

Reply:
I've had a look, but...

I don't think this change is worthwhile, since it necessitates us creating all the transaction type records in all the relevant unit and request specs —and I think the cost of maintaining the list of income types we want to display in the form is low.

If this is something you feel passionate about, we can add a ticket to the backlog and think about it outside the context of this PR.

How does that sound to you?